tests/test-node.py

```diff
@@ -54,7 +54,10 @@ def test_node_scontrol():
 
 def test_node_update():
     """Node: Test node().update()."""
-    node_test_before = pyslurm.node().find_id("c10")
+    import time
+    time.sleep(3)
```

Comment:
Why the 3 second sleep?

Reply:
Sorry, but does not really image, why there is a sleep, but I checked out my commit message and this sounds reasonable.
" Take the last node instead of c10 and add 3 seconds sleep for the scontrol show node test, 3 seconds sleep is needed otherwise the state of the node is still ALLOCATED"

Reply:
Ok, we can update this to a wait decorator in a separate PR

Reply:
If you know how, you could also split this in another PR. If I remember correctly, it did not work without waiting for the 3 seconds.